src/multisend.py

```diff
@@ -40,7 +42,7 @@ def build_encoded_multisend(
 
 def prepend_unwrap_if_necessary(
     client: EthereumClient,
-    safe_address: str | ChecksumAddress,
+    safe_address: ChecksumAddress,
```

Comment:
Note that this was the reason for the apparently unnecessary `to_checksum_address` commented [here](https://github.com/cowprotocol/solver-rewards/pull/309#discussion_r1252065532)